code/Commenting.php

```diff
@@ -13,12 +13,18 @@
  */
 
 class Commenting {
-	
+
 	/**
 	 * @var array map of enabled {@link DataObject} and related configuration
 	 */
 	private static $enabled_classes = array();
-	
+
+	/**
+	 * @config
+	 * @var bool Whether to enable commenting on SiteTree objects by default
+	 */
+	private static $sitetree_comments = true;
+
```

Comment:
But this is to allow you to prevent it being applied to sitetree by default, currently it gets applied to every site tree object in the module's _config.php so you can't prevent it, but often you might only want it on one page class (like BlogPage for example)